### DISCUSSION TITLE
v0.4.4

### DISCUSSION THREAD

**Discussion by owen-reorg** [bnb-chain/opbnb#230] [Announcements]
## v0.4.4

This release includes important fixes to help pbss geth nodes automatically recover from ungraceful shutdowns.
We recommend upgrading to this version if you are using the pbss mode.

### What's Changed
* fix(devnet): Modify the blob configuration of the bsc devnet by @welkin22 in https://github.com/bnb-chain/opbnb/pull/212
* feat: auto recover from pbss geth unclean shutdown by @krish-nr in https://github.com/bnb-chain/opbnb/pull/214
* fix: fix el bug by @krish-nr in https://github.com/bnb-chain/opbnb/pull/215

### Docker Images

- ghcr.io/bnb-chain/op-node:v0.4.4

**Full Changelog**: https://github.com/bnb-chain/opbnb/compare/v0.4.3...v0.4.4

<hr /><em>This discussion was created from the release <a href='https://github.com/bnb-chain/opbnb/releases/tag/v0.4.4'>v0.4.4</a>.</em>

**Comment by Issalk** [comment]
> ## v0.4.4
> 
> 
> 
> This release includes important fixes to help pbss geth nodes automatically recover from ungraceful shutdowns.
> 
> We recommend upgrading to this version if you are using the pbss mode.
> 
> 
> 
> ### What's Changed
> 
> * fix(devnet): Modify the blob configuration of the bsc devnet by @welkin22 in https://github.com/bnb-chain/opbnb/pull/212
> 
> * feat: auto recover from pbss geth unclean shutdown by @krish-nr in https://github.com/bnb-chain/opbnb/pull/214
> 
> * fix: fix el bug by @krish-nr in https://github.com/bnb-chain/opbnb/pull/215
> 
> 
> 
> ### Docker Images
> 
> 
> 
> - ghcr.io/bnb-chain/op-node:v0.4.4
> 
> 
> 
> **Full Changelog**: https://github.com/bnb-chain/opbnb/compare/v0.4.3...v0.4.4
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/bnb-chain/opbnb/releases/tag/v0.4.4'>v0.4.4</a>.</em>
